test/mpi/pt2pt/rqfreeb.c

```diff
@@ -100,16 +100,8 @@ int main(int argc, char *argv[])
                 fprintf(stderr, "message %d (%d) should be %d\n", i, rmsg[i], 10 + i);
             }
         }
-        /* The MPI standard says that there is no way to use MPI_Request_free
-         * safely with receive requests.  A strict MPI implementation may
-         * choose to consider these erroreous (an IBM MPI implementation
-         * does so)  */
-#ifdef USE_STRICT_MPI
-        MPI_Wait(&r[4], MPI_STATUS_IGNORE);
-#else
```

Comment:
So we determined that `MPI_Request_free` without `MPI_Wait` is standard compliant? Maybe replace it with a comment of such.

Reply:
Yes. Will add a comment.